lib/s3-pagable-stream.js

```diff
@@ -0,0 +1,43 @@
+var Readable = require('readable-stream').Readable;
+
+/**
+ * Returns a stream for any paged AWS function
+ * you can optionally provide a mapping function
+ * like S3::listObjectsV2()
+ *
+ * @param {function} req - a non executed AWS function
+ * @param {function} fn - a function that selects/maps the results
+ * @param {object} opts - stream options
+ */
+module.exports = function(req, fn, opts) {
+  opts = Object.assign({}, opts, { read: read, objectMode: true });
+  if (!fn)
+    fn = function(_in) {
```

Comment:
I would name it `identity` and move it at the top of the module

Reply:
I'm not following what you want here. You want that to be a named function called identity?

Reply:
Oh, I see. I thought about doing that.